entgql/annotation.go

```diff
@@ -4,7 +4,7 @@
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//      http://www.apache.org/licenses/LICENSE-2.0
+//     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
@tankbusta, there are many changes like this that are irrelevant to this PR. Would you mind reverting them so it will be easier to review? Thanks

cc @giautm